src/page/i-frame.page.ts

```diff
@@ -3,8 +3,25 @@ import { browser, by, element, ElementFinder, promise } from 'protractor';
 export class FramePage {
   private iframe1: ElementFinder;
 
+  private titleLabel: ElementFinder;
+
   constructor() {
     this.iframe1 = element(by.id('frame1'));
+    this.titleLabel = element(by.css('.main-header'));
+  }
+
+  public async switchToMainPage(): Promise<void> {
+    this.titleLabel = element(by.css('.main-header'));
+    await browser.switchTo().defaultContent();
+  }
+
+  public async switchToFrame(): Promise<void> {
+    this.titleLabel = element(by.id('sampleHeading'));
```

Comment:
as @germandavid85 suggests in [this PR](https://github.com/kliver98/protractor-workshop/pull/22#discussion_r673971118): 
We should have two page objects: one for the main page, and another for the iframe. Each with the corresponding title.